### PR TITLE
Respect active and in_navigation: fix bug 1128819

### PR DESCRIPTION
**Pull request by jgmize**
Add in_navigation to admin list_editable and list_display

### CODE REVIEW

**Comment by jgmize** [comment]
@glogiotatidis r?

File: masterfirefoxos/base/templates/base.html [hunk 1]
```diff
@@ -62,7 +62,9 @@ <h2 class="site-logo">
           <nav class="nav-main aside">
             <ul>
             {% for child in feincms_page.get_root().get_children() %}
```
Comment (glogiotatidis) [inline masterfirefoxos/base/templates/base.html]: `get_children()` returns a QuerySet so you could do `get_chidren().filter(active=True, in_navigation=True)` but this is also OK.

**Comment by glogiotatidis** [comment]
r+

File: masterfirefoxos/base/templates/base.html (hunk 1 above)
Comment (jgmize) [inline masterfirefoxos/base/templates/base.html]: good point, I'll update.

**Pull request merged**